styles/styles.css

```diff
@@ -716,6 +716,11 @@ main .section.light-blue {
   background-color: var(--transparent-blue-light-color);
 }
 
+/* section - padded */
+main .section.padded {
+  padding: 2rem !important;
```

Comment:
Please see if you can avoid `!important` by using a more specific selector than the one which you want to override.